DemoAppSwiftUI/DemoAppSwiftUIApp.swift

```diff
@@ -43,6 +43,7 @@ struct DemoAppSwiftUIApp: App {
         }
         .onChange(of: appState.userState) { newValue in
             if newValue == .loggedIn {
+                /*
```

Comment:
Will be added back later when sorting is consistent

Reply:
Might be good to add a comment in the code, but not a big deal 👍